NuGet.Services.Dashboard.Operations/Tasks/DashBoardTasks/AzureManagementTasks/CreateTrafficManagerStatusOverviewReportTask.cs

```diff
@@ -29,36 +29,48 @@ public override void ExecuteCommand()
             request.Headers.Add("x-ms-version: 2014-02-01");
             request.PreAuthenticate = true;
             request.Method = "GET";
-            WebResponse respose = request.GetResponse();
+            WebResponse response = request.GetResponse();
             //Schema of the response would be as specified in http://msdn.microsoft.com/en-us/library/azure/hh758251.aspx
-            Console.WriteLine(respose);
-            using (var reader = new StreamReader(respose.GetResponseStream()))
-            {               
+            using (var reader = new StreamReader(response.GetResponseStream()))
+            {
+                var responseContent = reader.ReadToEnd();
+                Console.WriteLine(responseContent);
+
                 XmlDocument doc = new XmlDocument();
-                doc.LoadXml(reader.ReadToEnd());
+                doc.LoadXml(responseContent);
                 XmlNodeList parentNode = doc.GetElementsByTagName("Endpoint", "http://schemas.microsoft.com/windowsazure");
-                List<Tuple<string, string>> endpointValues = new List<Tuple<string, string>>();
+                //End point Structure
+                //<Endpoint >
```

Comment:
Thank you for this.

It might make even more sense to add a comment describing that what we're doing in the next loop is iterating through a list of these nodes? I was very confused at first upon reading the code, thinking that we were iterating through the children of the Endpoint node here instead of a list of the Endpoint nodes, but just changing the names as I mentioned might be enough.